Send-Rest_li-Request-Query-In-Request-Body_converted.md

```diff
@@ -1,9 +1,12 @@
 ---
 layout: guide
-title: Send-Rest.li-Request-Query-In-Request-Body
+title: Send Rest.li request query in request Body
 permalink: /Send-Rest_li-Request-Query-In-Request-Body
+excerpt: Rest.li protocol specifies what HTTP method will be used for each type of Rest.li request. However, sometimes due to security constraint or jetty buffer limitation, we may want to customize the HTTP method used to send a particular Rest.li request to a Rest.li server.
```

Comment:
Third person:

Rest.li protocol specifies what HTTP method will be used for each type of Rest.li request. However, sometimes due to security constraint or jetty buffer limitation, _it may be required_ to customize the HTTP method used to send a particular Rest.li request to a Rest.li server.